scripts/pre_commit_linter.py

```diff
@@ -1771,7 +1793,8 @@ def main():
     """Main method for pre commit linter script that lints Python and JavaScript
     files.
     """
-    all_files = _get_all_files()
+    global _MODE # pylint: disable=global-statement
```

Comment:
Hi Sean, will it okay to use a global variable here? We are unable to think of any other alternative. Do you have any suggestions? Thanks!

Reply:
I wouldn't do that. Pass it as a boolean argument to the relevant functions as needed.

An alternative would be to create a LintChecksManager class and initialize it so that it has an internal "verbose" property. You can then make the individual lint functions into methods of that class, and those methods can access the "verbose" property. This might be better since you won't have to pass stuff all over the place in that case.

Reply:
Done